packages/react/src/ReactElement.js

```diff
@@ -237,10 +237,9 @@ export function createElement(type, config, children) {
         typeof props.$$typeof === 'undefined' ||
         props.$$typeof !== REACT_ELEMENT_TYPE
       ) {
-        var displayName =
```

Comment:
Isn't this weird though? I don't think I touched this code but now it wants me to reformat it.